src/rollback_to_stable/rts_history.c

```diff
@@ -54,8 +54,13 @@ __wt_rts_history_delete_hs(WT_SESSION_IMPL *session, WT_ITEM *key, wt_timestamp_
         if (hs_tw->stop_ts <= ts)
             break;
 
-        if (!dryrun)
+        if (!dryrun) {
+            __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
+              WT_RTS_VERB_TAG_HS_UPDATE_ABORT "deleting history store update with stop_ts=%" PRIu64,
```

Comment:
Was `s_all` happy with this? It usually puts the macros on their own line. In any case, the start of the message is missing a capital.

Reply:
Update: it's been pointed out to me that most of the current RTS messages start with lowercase, so it's just `s_all` to worry about here.

Reply:
Done! [79d4acf](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/79d4acfc889e839293ada7ada4ba6b187868d65d)